scripts/init.sh

```diff
@@ -24,10 +24,34 @@ for var in "${vars_to_check[@]}"; do
   export "${var}"
 done
 
+# Setting the Node Role
+env_file_exist "${ROOT_DIR}/${ENV_FILE}"
+# shellcheck disable=SC1090
+source "${ROOT_DIR}/${ENV_FILE}" || { echo "Error: could not source ${ROOT_DIR}/${ENV_FILE} file. Fix it before proceeding any further.  Exiting..."; exit 1; }
+
+if [ -z "${SCNODE_ROLE:-}" ]; then
+  read -rp "Which kind of node would you like to start ? Choose between 'rpc' or 'forger': " SCNODE_ROLE_VALUE
+  SCNODE_ROLE="${SCNODE_ROLE_VALUE}"
+  while [[ ! "${SCNODE_ROLE_VALUE}" =~ ^(rpc|forger)$  ]]; do
+    echo -e "Error: The options have to be 'rpc' or 'forger'. Try again...\n"
+    read -rp "Which kind of node would you like to start ? Choose between 'rpc' or 'forger': " SCNODE_ROLE_VALUE
+    SCNODE_ROLE="${SCNODE_ROLE_VALUE}"
+  done
+  sed -i'' "s/SCNODE_ROLE=/SCNODE_ROLE=${SCNODE_ROLE}/g" ${ROOT_DIR}/${ENV_FILE}
```

Comment:
Why do you need '' single empty quotes right after sed -i word? :thinking:

Reply:
Those two single empty quotes are for avoiding troubles with MacOs sed command. Overwriting the previously set value should not be necessary because the condition of the if is true only when SCNODE_ROLE is not set I guess. Anyway I'll add it because it's safer.